vars/neoDeploy.groovy

```diff
@@ -1,94 +1,85 @@
 
         // Backward compatibility: ensure old configuration is taken into account
         // The old configuration in not stage / step specific
 
         def defaultDeployHost = script.commonPipelineEnvironment.getConfigProperty('DEPLOY_HOST')
         if(defaultDeployHost) {
-            echo "[WARNING][${stepName}] A deprecated configuration framework is used for configuring parameter 'DEPLOY_HOST'. This configuration framework will be removed in future versions."
-            stepConfiguration.put('host', defaultDeployHost)
+            echo "[WARNING][${STEP_NAME}] A deprecated configuration framework is used for configuring parameter 'DEPLOY_HOST'. This configuration framework will be removed in future versions."
+            stepCompatibilityConfiguration.put('host', defaultDeployHost)
         }
 
         def defaultDeployAccount = script.commonPipelineEnvironment.getConfigProperty('CI_DEPLOY_ACCOUNT')
         if(defaultDeployAccount) {
-            echo "[WARNING][${stepName}] A deprecated configuration framework is used for configuring parameter 'DEPLOY_ACCOUNT'. This configuration framekwork will be removed in future versions."
-            stepConfiguration.put('account', defaultDeployAccount)
+            echo "[WARNING][${STEP_NAME}] A deprecated configuration framework is used for configuring parameter 'DEPLOY_ACCOUNT'. This configuration framekwork will be removed in future versions."
+            stepCompatibilityConfiguration.put('account', defaultDeployAccount)
         }
 
         if(parameters.deployHost && !parameters.host) {
-            echo "[WARNING][${stepName}] Deprecated parameter 'deployHost' is used. This will not work anymore in future versions. Use parameter 'host' instead."
+            echo "[WARNING][${STEP_NAME}] Deprecated parameter 'deployHost' is used. This will not work anymore in future versions. Use parameter 'host' instead."
             parameters.put('host', parameters.deployHost)
         }
 
         if(parameters.deployAccount && !parameters.account) {
-            echo "[WARNING][${stepName}] Deprecated parameter 'deployAccount' is used. This will not work anymore in future versions. Use parameter 'account' instead."
+            echo "[WARNING][${STEP_NAME}] Deprecated parameter 'deployAccount' is used. This will not work anymore in future versions. Use parameter 'account' instead."
             parameters.put('account', parameters.deployAccount)
         }
 
         def credId = script.commonPipelineEnvironment.getConfigProperty('neoCredentialsId')
-
         if(credId && !parameters.neoCredentialsId) {
-            echo "[WARNING][${stepName}] Deprecated parameter 'neoCredentialsId' from old configuration framework is used. This will not work anymore in future versions."
+            echo "[WARNING][${STEP_NAME}] Deprecated parameter 'neoCredentialsId' from old configuration framework is used. This will not work anymore in future versions."
```

Comment:
Why do we emit a warning here only and do not use the ```stepCompatibilityConfiguration``` like above?

Reply:
see above